### ISSUE TITLE
Waiting for operation to complete indicator

### ISSUE THREAD

**Issue by dmgursky** [gramps-project/Gramps.js#238]
When waiting for greampsweb to complete an operation, the bar under the title banner goes to a rotating light (of sorts).  How ever, because this bar is not especially deep, it is difficult to notice, especially on newer, high density monitors.  Please increase the depth of this bar (or make the depth selectable from user preferences, as different users will have different size monitors

**Comment by dmgursky** [comment]
(Errata, grampsweb, not greampsweb)

**Comment by DavidMStraub** [comment]
I don't think it has anything to do with monitor resolution. High DPI displays will use a different device pixel ratio. The progress bar has a fixed CSS pixel height which scales with everything else, including text size. That it's subtle is intentional. I don't see the point in distracting the user with a fat progress bar. Compare this e.g. to Firefox

![grafik](https://github.com/gramps-project/Gramps.js/assets/10965193/3127b151-f879-422d-bf96-f2652f8d2332)

In short, I think it's wontfix, but please feel free to share a screenshot what it looks like in your browser, could also be a browser quirk.

**Comment by dmgursky** [comment]
I'm an old fogey who is unaccustomed to these fancy doohickeys.  NOW
WHERE'S MY ASCII ART!  :-D

*David Gursky • Washington, DC*
[EMail ***@***.***>] *•* [Facebook
<https://www.facebook.com/dmgursky>] *•* [LinkedIn
<https://www.linkedin.com/in/dgursky/>]
